### PR TITLE
Improve code

### LINKED ISSUE #16: Handling of price_if_smaller_else()
```
def price_if_smaller_else(price, price_smaller, else_min, prices_comp, c):
    return(price if min(prices_comp[comp] for comp in range(numberPlayers) if comp!=c)< price_smaller else round(min([max(else_min, prices_comp[company]-0.5) for company in range(numberPlayers) if company != c]),2))
```

First, this is a function that's part of the shared dictionaries S1-S3. So we'll might want to create a very simple module to include in each notebook, that contains such shared methods.

But more importantly: Tom, please rewrite for better readability. It's a mess. :)

### PR DESCRIPTION
**Pull request by tomschw**
Reformat code for better readability.
Resolve #16

### CODE REVIEW

File: monte_carlo.ipynb [hunk 1]
```diff
@@ -34,18 +34,17 @@
...
-    "numberOfCustomerEvents = round(1.1*timeHorizon/meanTimeBetweenCustomerArrival)\n",
+    "num_firm = 5\n",
+    "time_horizon = 1\n",
+    "num_market_events = round(1.1*time_horizon)\n",
+    "mean_betw_customer_arrival = 2\n",
     "prices = np.arange(0,15.1,0.2)   # values used for paper: (np.arange(0,15.1,0.1))\n",
-    "watchedCompany = 0\n",
-    "numberSimulations = 100  # value used for paper: 10000\n",
-    "numberConsideredMarketSituations = 10   # value used for paper: 40\n",
+    "watched_firm = 0\n",
```
Comment (Bouncner) [inline monte_carlo.ipynb]: Could you add short comments here?

File: monte_carlo.ipynb [hunk 2]
```diff
@@ -34,18 +34,17 @@
...
+    "num_sims = 100  # value used for paper: 10000\n",
+    "num_market_situations = 10   # value used for paper: 40\n",
```
Comment (Bouncner) [inline monte_carlo.ipynb]: What does this variable do?

File: intensity.ipynb [hunk 3]
```diff
@@ -55,9 +54,9 @@
    },
    "outputs": [],
    "source": [
-    "pricesCompanies = np.array([round(random.uniform(6,16),2) for _ in range(numberPlayers)])\n",
-    "qualityCompanies = np.array([round(random.uniform(0.5,5.5)) for _ in range(numberPlayers)])\n",
-    "ratingCompanies = np.array([round(random.uniform(90,100),1) for _ in range(numberPlayers)])"
+    "prices_firm = np.array([\n",
```
Comment (Bouncner) [inline intensity.ipynb]: Why no numpy uniform here?

File: intensity.ipynb [hunk 4]
```diff
@@ -104,76 +109,167 @@
...
-    "\n",
-    "def calculatePrices(pricesCompaniesAdjustments, adjustingFirm, strategy):\n",
-    "    pricesCompaniesAdjustments = np.array(pricesCompaniesAdjustments)\n",
-    "    for i in range(1,len(pricesCompaniesAdjustments)-1):        \n",
-    "        pricesCompaniesAdjustments[i:,adjustingFirm[i]] = strategy[str(adjustingFirm[i])](pricesCompaniesAdjustments[i-1],adjustingFirm[i])\n",
-    "    pricesCompaniesAdjustments[len(pricesCompaniesAdjustments)-1] = pricesCompaniesAdjustments[len(pricesCompaniesAdjustments)-2]\n",
-    "    return pricesCompaniesAdjustments"
+    "def calc_prices(prices_firm_adjust, adjusting_firm, strategy):\n",
+    "    prices_firm_adjust = np.array(prices_firm_adjust)\n",
+    "    for i in range(1,len(prices_firm_adjust)-1):        \n",
```
Comment (Bouncner) [inline intensity.ipynb]: Short comment, why having this 'strange' indexing in the for loop.

**Review by Bouncner — CHANGES_REQUESTED** [review]
First batch of short comments. In general: please add more helping comments.

Besides that: looks much better now!

**Comment by tomschw** [comment]
@Bouncner Updated each file which should answer all your questions.
I also changed the price_if_smaller_else function (#16), is it better?

**Pull request merged**